### DISCUSSION TITLE
有安装成功的吗？二进制包在Deepin23无法安装呀？

### DISCUSSION THREAD

**Discussion by italks** [linuxdeepin/developer-center#5510] [General | 常规讨论]
![image](https://github.com/linuxdeepin/developer-center/assets/1667764/f55d095a-5dad-4e93-a3bb-d5ab55346aac)
![image](https://github.com/linuxdeepin/developer-center/assets/1667764/7c60ed16-c182-4b20-b982-f9c34ecc60a5)

**Comment by BLumia** [comment]
感觉这个包像是给 v20 打的吧？

cc @deepin-mozart

**Reply by deepin-mozart** [reply]
是的，v23暂未提安装包，后续会上传

**Comment by ice909** [comment]
我在23上打了一个包，你试试？
包有点大，github无法上传
